client/src/layouts/header/searchForm/SearchBar/SearchBar.jsx

```diff
@@ -1,14 +1,20 @@
 import React from 'react';
 import classes from './SearchBar.module.css';
 import searchIcon from '../../../../../public/images/searchIcon.png';
+import PropTypes from 'prop-types'
 
-function SearchBar({ theme, searchText, handleSearchText }) {
+function SearchBar({ searchText, handleSearchText }) {
   return (
-    <form className={classes.container} data-theme={theme}>
+    <form className={classes.container}>
```

Comment:
Why do you need form ?

Reply:
coz i have input, i wrote in firsts prs without it, but had a comment, that i need to add a form tag